modules/uhttp/filters.go

```diff
@@ -55,8 +55,7 @@ type contextFilter struct {
 }
 
 func (f contextFilter) Apply(w http.ResponseWriter, r *http.Request, next http.Handler) {
-	ctx := ulog.NewLogContext(r.Context())
-	next.ServeHTTP(w, r.WithContext(ctx))
+	next.ServeHTTP(w, r)
```

Comment:
Probably. If it's okay with you, I'd rather keep this PR restricted to the logging. I'm going to be rebasing endlessly as it is.